lighthouse-core/gather/driver.js

```diff
@@ -849,6 +849,24 @@ class Driver {
     });
   }
 
+  /**
+   * @param {number} [timeout]
+   * @return {Promise<LH.Crdp.Security.SecurityStateChangedEvent>}
+   */
+  getSecurityState(timeout = 1000) {
+    return new Promise((resolve, reject) => {
+      const err = new LHError(LHError.errors.SECURITY_STATE_TIMEOUT);
+      const asyncTimeout = setTimeout((_ => reject(err)), timeout);
+
+      this.once('Security.securityStateChanged', state => {
```

Comment:
does this check break if someone else has been listening to the `Security` domain? It would be nice to get a non-event based version of this in the protocol

Reply:
I enabled just before calling this function, and yes, it does break. It hangs, and eventually rejects with `SECURITY_STATE_TIMEOUT` (which actually kills Lighthouse. oops)

Besides other users enabling the Security domain, could this fail if a developer opens the Security tab (this enables the domain) before running an audit?